purchases/src/main/kotlin/com/revenuecat/purchases/customercenter/events/CustomerCenterEvent.kt

```diff
@@ -0,0 +1,45 @@
+package com.revenuecat.purchases.customercenter.events
+
+import com.revenuecat.purchases.ExperimentalPreviewRevenueCatPurchasesAPI
+import com.revenuecat.purchases.utils.serializers.DateSerializer
+import com.revenuecat.purchases.utils.serializers.UUIDSerializer
+import kotlinx.serialization.Serializable
+import java.util.Date
+import java.util.UUID
+
+/**
+ * Type representing a customer center event and associated data. Meant for RevenueCatUI use.
+ */
+@ExperimentalPreviewRevenueCatPurchasesAPI
+@Serializable
+data class CustomerCenterEvent(
+    val creationData: CreationData,
+    val eventData: Data,
+) {
+
+    @Serializable
+    data class CreationData(
+        @Serializable(with = UUIDSerializer::class)
+        val id: UUID,
+        @Serializable(with = DateSerializer::class)
+        val date: Date,
+    )
+
+    @Serializable
+    data class Data(
+        @Serializable(with = UUIDSerializer::class)
+        val id: UUID?,
```

Comment:
Hmm what does this id represent/how does it differ from the one in the `CreationData`? We might want to give it a different name if needed... And depending on the answer to that, is it ok for it to be nullable?

Reply:
Actually, I am not sure. Just double-checked, and it does not make any sense. Removed it 👍